README.md

```diff
@@ -81,16 +93,6 @@ xbps-install alacritty
    rustup update stable
    ```
 
-#### Ubuntu
-
-On Ubuntu, you need a few extra libraries to build Alacritty. Here's an `apt`
-command that should install all of them. If something is still found to be
-missing, please open an issue.
-
-```sh
-apt-get install cmake libfreetype6-dev libfontconfig1-dev xclip
-```
-
```

Comment:
These are still revelant for people trying to build alacritty on Ubuntu, I don't think removing this section is a good idea.